aopy/analysis.py

```diff
@@ -110,6 +110,32 @@ def get_pca_dimensions(data, max_dims=None, VAF=0.9, project_data=False):
 
     return list(explained_variance), num_dims, projected_data
 
+def estimate_PR(data, method = 'eig'):
```

Comment:
I wonder if the default should be calculated using the trace since I think it is more computationally efficient. If we use the 'eig' method we should also probably return the eigen vectors.

Reply:
Hi Ryan, I made some code changes to the code. I removed 'trace' as a method in my analysis code just to stick to one method that works all the time. I ran into some complex numbers issue with 'trace' a couple of time. I know the current code works.